sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/http/HttpTimeoutPolicyDefault.java

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos.implementation.http;
+
+import com.azure.cosmos.implementation.Configs;
+import io.netty.handler.codec.http.HttpMethod;
+
+
+import java.time.Duration;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+public class HttpTimeoutPolicyDefault extends HttpTimeoutPolicy {
+
+    public static final HttpTimeoutPolicy INSTANCE = new HttpTimeoutPolicyDefault();
+
+    private HttpTimeoutPolicyDefault() {
+    }
+
+    @Override
+    public long maximumRetryTimeLimit() {
+        return Configs.getHttpResponseTimeoutInSeconds();
+    }
+
+    @Override
+    public List<ResponseTimeoutAndDelays> getTimeoutList() {
+        return getTimeoutAndDelays();
+    }
+
+    // Assume that it is not safe to retry unless it is a get method.
+    // Create and other operations could have succeeded even though a timeout occurred.
+    @Override
+    public boolean isSafeToRetry(HttpMethod httpMethod) {
+        return httpMethod == HttpMethod.GET;
+    }
+
+    private List<ResponseTimeoutAndDelays> getTimeoutAndDelays() {
```

Comment:
We can remove this API, and directly override `getTimeoutList()` API above.

Reply:
Fixed